src/OrchardCore.Modules/OrchardCore.Forms/Models/FormElementLabelPart.cs

```diff
@@ -0,0 +1,15 @@
+using OrchardCore.ContentManagement;
+
+namespace OrchardCore.Forms.Models;
+
+public class FormElementLabelPart : ContentPart
+{
+    public LabelOptions Option { get; set; }
+
+    public string Label { get; set; }
+}
+
+public class FormElementValidationPart : ContentPart
+{
+    public ValidationOptions Option { get; set; }
```

Comment:
Set Default to None so exiting content item won't break with NRE

Reply:
@ns8482e `ValidationOptions` is an enum with the default value of `None` so it's always `None` unless a user changes it. so it wont break anything